src/arrayChecks.js

```diff
@@ -0,0 +1,165 @@
+const assert = require('assert');
+const assertInRange = require('./numberChecks').assertInRange;
+const assertInRangeL = require('./numberChecks').assertInRangeL;
+const assertInRangeR = require('./numberChecks').assertInRangeR;
```

Comment:
There's a shorthand for that:
```suggestion
const { assertInRange, assertInRangeL, assertInRangeL } = require('./numberChecks');
```